resources/eventing/charts/event-publisher-nats/templates/service.yaml

```diff
@@ -21,7 +21,12 @@ spec:
   type: ClusterIP
   selector: {{- include "event-publisher-nats.selectorLabels" . | nindent 4 }}
   ports:
-    - protocol: TCP
+    - name: metrics
+      protocol: TCP
       port: 80
       targetPort: {{ .Values.metrics.service.port }}
+    - name: proxy-status
```

Comment:
IMO we should not use metrics endpoint for liveness/readiness. There are already [readyz](https://github.com/kyma-project/kyma/blob/master/resources/eventing/charts/event-publisher-nats/templates/deployment.yaml#L53) and health(https://github.com/kyma-project/kyma/blob/master/resources/eventing/charts/event-publisher-nats/templates/deployment.yaml#L43) endpoints for the same purpose. We can create a new svc with added istio configuration for these endpoints to make them available from outside the mesh.